HelpSource/Tutorials/Streams-Patterns-Events7.schelp

```diff
@@ -74,7 +74,7 @@ SynthDef(\help_SPE4_CFString2, { arg i_out, freq = 360, gate = 1, pan, amp=0.1,
 	out = eg * RLPF.ar(osc, fc, 0.1);
 	#a, b = out;
 	Out.ar(i_out, Mix.ar(PanAz.ar(4, [a, b], [pan, pan+0.3])));
-}).send(s);	// change .send(s) to .add
+}).add
```

Comment:
please keep send, it is part of the explanation!

Reply:
Thank you, and my apologies for the change. I have reverted it in https://github.com/supercollider/supercollider/pull/7117/commits/2428a7dca08f93f3d32973e0045a74d42aaeff12.